docs/targets/googlesheets.md

```diff
@@ -1,42 +1,43 @@
 # Event Target for Google Sheets
 
-This event target receives [CloudEvents][ce] over HTTP and appends the event payload to a GoogleSheets sheet.
+This event Target receives [CloudEvents][ce] over HTTP and appends the event payload to a GoogleSheets Sheet.
 
-## Prerequisites
+## Prerequisite(s)
 
-1. Google API Credentials
-1. GoogleSheets Sheet id
+- Google API Credentials
+- GoogleSheets Sheet ID
 
-### Google API Credentials
+Consult the [Secrets](../guides/secrets.md) guide for more information about
+how to add the Google API key as a secret.
 
-1. Head to [Google Developers Console][google-dashboard] and create a new
- project (or select the one you have.)
-2. Under “**APIs & Services > Library**”, search for “**Sheets API**” and enable it.
-3. Go to “**APIs & Services > Credentials**” and choose “**Create credentials > Service account**”. (No extra roles nor
- users access is required, you can skip optional fields)
-4. On the last step of service account creation download the JSON key.
+## Google API Credentials
 
-Use the `client_email` field within the credentials JSON file you downloaded to share the
-Google Spreadsheets you want the Target to have access to. (when sharing Notifications mark should be disabled)
+1. Head to the [Google Developers Console][google-dashboard] and create a new
```

Comment:
I know you were just following the provided, I just learned that if you number these all as 1., it will number them correctly. That way if we ever have to add or take away something, it will auto number and the dev will not need to change the numbering.